src/mpid/ch4/generic/gpu/mpidig_gpu_utils.h

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Copyright (C) by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+#ifndef MPIDIG_GPU_UTIL_H_INCLUDED
+#define MPIDIG_GPU_UTIL_H_INCLUDED
+
+#include "ch4_impl.h"
+
+/* MPIDIG_gpu_stage_copy_D2H - moving data from device buffer to stage host buffer. */
+MPL_STATIC_INLINE_PREFIX void MPIDIG_gpu_stage_copy_D2H(const void *sbuf, void *dbuf,
```

Comment:
Probably better to keep lower case for `d2h`.

Reply:
done